stable/traefik/README.md

```diff
@@ -93,6 +93,7 @@ The following table lists the configurable parameters of the Traefik chart and t
 | `loadBalancerSourceRanges`      | list of IP CIDRs allowed access to load balancer (if supported)      | None                                      |
 | `externalTrafficPolicy`         | Set the externalTrafficPolicy in the Service to either Cluster or Local | `Cluster`                              |
 | `replicas`                      | The number of replicas to run; __NOTE:__ Full Traefik clustering with leader election is not yet supported, which can affect any configured Let's Encrypt setup; see Clustering section | `1` |
+| `minAvailable`                  | minimum number of available pods for PodDisruptionBudget             | `1` |
```

Comment:
imo, the name of the option, `minAvailable`, is not adequately self-descriptive. "minAvailable" doesn't provide sufficient context. Minimum what? For what? The documentation is good, but the option name itself should provide context.

Reply:
`minAvailablePods` then? Or make it something like `pdb.minAvailable`, so it's clear that this is the `minAvailable` option of the PDB?

Reply:
yeah, I'm not sure it's a good name either, I was just following the pattern: https://github.com/kubernetes/charts/search?utf8=%E2%9C%93&q=minavailable&type=

Reply:
Ah. I didn't realize this was an established pattern.

Let's ask someone who's more involved across a wider sampling of charts whether this is a convention that should be observed.

@prydonius or @unguiculus maybe one of you gentlemen could weigh in?

Reply:
I wouldn't say it's established and I'd like to advise for something like this, so users have the option whether they want `minAvailable` or `maxUnavailable`.

https://github.com/kubernetes/charts/blob/master/stable/mongodb-replicaset/templates/mongodb-poddisruptionbudget.yaml#L16

Reply:
PR updated :)